src/test/java/com/fasterxml/jackson/databind/MapperFootprintTest.java

```diff
@@ -0,0 +1,18 @@
+package com.fasterxml.jackson.databind;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openjdk.jol.info.GraphLayout;
+
+public class MapperFootprintTest {
+    @Test
+    @Ignore
```

Comment:
Probably remove this?

Reply:
see comment below

Reply:
Ah. Another thing we can do to avoid running as part of test suite (but to allow manual run) would be to move under failing. But we can figure out best way later on.